charts/k8s-watcher/values.yaml

```diff
@@ -113,6 +113,8 @@ watcher:
     csiNode: true
     csiStorageCapacity: true
     volumeAttachment: true
+    allowReadAll: true
+    customAPIGroups: []
```

Comment:
It is unclear how it should be formatted + need to add documentation on the values table in the md file https://github.com/komodorio/helm-charts/blob/master/charts/k8s-watcher/README.md